massa-protocol-worker/src/protocol_worker.rs

```diff
@@ -996,6 +1021,22 @@ impl ProtocolWorker {
             self.note_operations_to_announce(&to_announce, op_timer)
                 .await;
 
+            // send Operation(Denunciation) to DenunciationFactory
+            let de_interest_ops = new_operations
+                .iter()
+                .filter(|(_, wrapped_op)| {
+                    matches!(wrapped_op.content.op, OperationType::Denunciation { .. })
+                })
+                .map(|(_, wrapped_op)| wrapped_op.clone())
```

Comment:
You can use a `filter_map()` here instead of `map` then `filter`

Reply:
done